server/sonar-server/src/test/java/org/sonar/server/search/FacetsMediumTest.java

```diff
@@ -136,6 +135,7 @@ public void should_process_result_with_date_histogram() throws Exception {
     SearchRequestBuilder search = esTester.client().prepareSearch(INDEX).setTypes(TYPE)
       .addAggregation(
         AggregationBuilders.dateHistogram(FIELD_CREATED_AT)
+          .minDocCount(0L)
```

Comment:
the default value is changed in ES 2.x, so it is enforced here. Note that the test is still incomplete as the default value is 1 in ES 1.7 and test does not fail when enforcing 0. The test should be refactored.